core/App/screens/CredentialOffer.tsx

```diff
@@ -62,8 +63,10 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
   const handleAcceptPress = async () => {
     try {
       setAcceptModalVisible(true)
-
-      await agent.credentials.acceptOffer(credential.id)
+      await agent.credentials.acceptOffer({
+        credentialRecordId: credential.id,
+        autoAcceptCredential: AutoAcceptCredential.ContentApproved,
```

Comment:
Is this necessary given the `AcceptOfferOptions` type doesn't include `autoAcceptCredential`?